celestia-app/private/config/config.toml

```diff
@@ -275,7 +275,7 @@ dial_timeout = "3s"
 # Mempool version to use:
 #   1) "v0" - (default) FIFO mempool.
 #   2) "v1" - prioritized mempool.
-version = "v0"
+version = "v1"
```

Comment:
Why different mempool?

Reply:
I actually didn't purposefully change this, it's what was put in by default when generating genesis. Obviously, the generated text doesn't match.